nl/mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -1,26 +1,26 @@
-<!ENTITY aboutSupport.appBasicsAppBuildID "Application Build ID">
+<!ENTITY aboutSupport.appBasicsAppBuildID "Applicatie Build ID">
 
-<!ENTITY aboutSupport.accountsTitle "Mail and News Accounts">
-<!ENTITY aboutSupport.showPrivateData.mainText "Include account names">
-<!ENTITY aboutSupport.showPrivateData.explanationText "(possibly identifying information)">
+<!ENTITY aboutSupport.accountsTitle "Mail en Nieuws Accounts">
+<!ENTITY aboutSupport.showPrivateData.mainText "Inclusief account namen">
```

Comment:
CORRECTION NEEDED.
- ''account namen'' should be ''accountnamen''. 
- ''Inclusief'' should be ''Includeer'', depending on context.

New suggestion depending on context: ''Includeer accountnamen".